recipes/ixwebsocket/all/conanfile.py

```diff
@@ -41,33 +39,33 @@ def _can_use_openssl(self):
         return True
 
     def configure(self):
-        if self.options.use_mbed_tls and not self.options.use_tls or self.options.get_safe("use_openssl") and not self.options.use_tls:
-            raise ConanInvalidConfiguration("TLS must be enabled to use mbedtls")
-        elif self.options.use_mbed_tls and self.options.get_safe("use_openssl"):
-            raise ConanInvalidConfiguration("Cannot use both OpenSSL and MbedTLS")
+        if self.options.tls == "applessl" and not tools.is_apple_os(self.settings.os):
+            raise ConanInvalidConfiguration("Can only use Apple SSL on Apple.")
+        elif self.options.tls and not self.options.use_tls:
+            raise ConanInvalidConfiguration("You cannot use an SSL system when use_tls is false")
+        elif not self.options.tls and self.options.use_tls:
+            raise ConanInvalidConfiguration("You need to select an SSL system when use_tls is true")
```

Comment:
Given the two conditions before this line, I wonder if the `use_tls` option is needed. The value `None` for `tls` can work the same as "use_tls=False", wdyt?

Reply:
Yeah, that is reasonable. I'll get rid of  `use_tls`